packages/lib/gpcircuits/test/virtual-entry.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { POD, decodePublicKey } from "@pcd/pod";
+import { BABY_JUB_NEGATIVE_ONE } from "@pcd/util";
+import { WitnessTester } from "circomkit";
+import "mocha";
+import { poseidon2 } from "poseidon-lite";
+import {
+  CircuitSignal,
+  VirtualEntryModuleInputNamesType,
+  VirtualEntryModuleInputs,
+  VirtualEntryModuleOutputNamesType,
+  VirtualEntryModuleOutputs,
+  array2Bits,
+  padArray
+} from "../src";
+import { circomkit, privateKey, privateKey2, sampleEntries } from "./common";
+
+describe("Virtual entry module should work", function () {
+  // Circuit compilation sometimes takes more than the default timeout of 2s.
+  let circuit: WitnessTester<
+    VirtualEntryModuleInputNamesType,
+    VirtualEntryModuleOutputNamesType
+  >;
+
+  function makeTestSignals(
+    paramMaxObjects: number,
+    isValueHashRevealedBits: CircuitSignal[]
+  ): {
+    inputs: VirtualEntryModuleInputs;
+    outputs: VirtualEntryModuleOutputs;
+  } {
+    const paddedRevealedBits = padArray(
+      isValueHashRevealedBits,
+      paramMaxObjects,
+      isValueHashRevealedBits[0]
```

Comment:
Why is this the pad?  I'd expect for an isRevealed value the pad would be 0

Reply:
Arbitrary choice. 0 makes sense though.